cranelift/codegen/src/ir/dfg.rs

```diff
@@ -45,6 +45,41 @@ impl IndexMut<Inst> for Insts {
     }
 }
 
+/// Storage for basic blocks within the DFG.
+#[derive(Clone, PartialEq, Hash)]
+#[cfg_attr(feature = "enable-serde", derive(Serialize, Deserialize))]
+pub struct Blocks(PrimaryMap<Block, BlockData>);
+
+impl Blocks {
+    /// Create a new basic block.
+    pub fn make_block(&mut self) -> Block {
```

Comment:
Bikeshed: Since `make_block` is already accessed through a field named "blocks", how about a name that doesn't have "block" in it? I was tempted to suggest `blocks.new()` but that's not exactly idiomatic, so maybe just `blocks.add()`?

Reply:
I like that!